src/components/global/TextInput.jsx

```diff
@@ -1,9 +1,12 @@
+import React from "react";
+
 function TextInput(props) {
   const { label, textValue, placeholderValue, onChangeHandler } = props;
   
   return (
     <div className="text-input">
       {label && 
+        // eslint-disable-next-line jsx-a11y/label-has-associated-control
```

Comment:
Why do we need this disable?

Reply:
I disabled it/didn't fix it as I don't know the context of this component (not being used yet). I will leave it disabled for now, @Angstylemon if you have free time you can look into this, cheers.